cli/zone.go

```diff
@@ -93,7 +92,7 @@ func runGetZone(cmd *cobra.Command, args []string) {
 		log.Errorf("Object %d: no zone config found", id)
 		return
 	}
-	fmt.Fprintln(osStdout, rows[0][1])
+	fmt.Println(rows[0][1])
```

Comment:
Removed all uses of `osStdout`. I'm passing a `io.Writer` instead, as is standard. It makes the sql_util functions a bit longer, but it's still preferable.